### PR TITLE
feat(office365): Add Office365 Message Trace.

### PR DESCRIPTION
**Pull request by PierrickV**

### CODE REVIEW

File: docs/xdr/features/collect/integrations/cloud_and_saas/office365/message_trace.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "StartDate": "/Date(1658851300604)/",
+        "EndDate": "/Date(1659024100604)/",
+        "Index": 0
+      }
+    ]
+  }
+}
+```
+
+You can now create enable the trigger "Create a new playbook" > "Create a playbook from scratch" and add the "Message Trace" trigger.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/office365/message_trace.md:52]: Maybe explain how to fill the configuration of the trigger.
It should be great to have the "Create intake" section too.

**Review by squioc — CHANGES_REQUESTED** [review]

File: docs/xdr/features/collect/integrations/cloud_and_saas/office365/message_trace.md (hunk 1 above)
Comment (PierrickV) [reply docs/xdr/features/collect/integrations/cloud_and_saas/office365/message_trace.md]: Hi @squioc, thank you for your review.

I applied your suggestions

**Comment by squioc** [comment]
Please, update mkdocs.yml to add your documentation in the menu

**Review by yosradirgham — DISMISSED** [review]
LGTM

**Review by squioc — APPROVED** [review]

**Comment by squioc** [comment]
Maybe add the expected Office 365 subscription to be able to activate the Message Trace in the documentation

**Review by squioc — APPROVED** [review]

**Pull request merged**